components/formats-gpl/src/loci/formats/in/FEITiffReader.java

```diff
@@ -350,6 +393,12 @@ else if (key.equals("User")) {
         else if (key.equals("Magnification")) {
           magnification = new Double(value);
         }
+        else if (key.equals("X") && "PixelsSize".equals(parent)) {
+          sizeX = new Double(value);
```

Comment:
For the QA file described in this PR, the main issue is that the physical pixel size seems to be expressed in meters. Since the format mostly deals with TEM data, the value is smaller than `Constants.EPSILON` and thus is registered as a zero value length. Thoughts about actively convert this value to microns (for Titan data at least)?